agent.go

```diff
@@ -853,6 +853,14 @@ func (s *sandbox) stopGRPC() {
 	}
 }
 
+func (s *sandbox) syncTimeHw2Sys() error {
+	cmd := &exec.Cmd{
+		Path: "/sbin/hwclock",
```

Comment:
heh - do we know we have this in all our rootfs distro variants btw?

Reply:
oh, yeah, you are right, I'm missing the osbuilder PR! Will send it out soon.

Reply:
Sent github.com/kata-containers/osbuilder/pull/213 and updated the `Depends-on` tag here.